src/sections/Company/News-single/Sidebar.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components";
+import Button from "../../../reusecore/Button";
+import { FiDownloadCloud } from "@react-icons/all-files/fi/FiDownloadCloud";
+import { FcSms } from "@react-icons/all-files/fc/FcSms";
+import { Link } from "gatsby";
+
+const NewsSidebarWrapper = styled.div`
+  
+Button{
+    margin: 0 0 1rem ;
+    @media screen and (max-width: 768px) {
+        display: block;
+        margin: 1rem auto;
+        }
+}
+.icon{
+    display: block;
+    margin: auto;
+}
+
+.project__block__inner { 
+    background: ${props => props.theme.white};
+    transition: 450ms all;
+    border: 1px solid ${props => props.theme.shadowLightColor};
+    &:hover{
+        box-shadow: 0 2px 10px rgba(0, 0, 0, 0.4);
+    }
+    padding: 12% 6% 12% 6%;
+    border-radius: 4%;
+    text-align: center;
+    h5{
+        font-weight: 700;
+        color: ${props => props.theme.secondaryColor}
+    }
+    p{
+        font-weight: 300;
+        color: ${props => props.theme.black};
+    }
+    @media screen and (max-width: 768px) {
+        margin: 6% 15%;
+        padding: 6% 10%;
+        }
+}
+
+`;
+
+const NewsSidebar = (props) => {
+  return (
+    <NewsSidebarWrapper>
+      {props.kit ? 
+        <a href={props.kit}>
+          <Button primary title="Get the Press Kit" external={true}>
+            <FiDownloadCloud size={21} className="icon-left" />
+          </Button>
+        </a> : "" }
+      <Link to="/brand">
+        <Button primary title="Check out our Brand Kit" external={true} />
+      </Link>
+      <a href="mailto:press@layer5.io" target="_blank" rel="noreferrer">
+        <div className="project__block__inner">
+          <h5>Media Contacts</h5>
+          <p>Kate Suttons</p>
+          <FcSms size={40} className="icon" />
```

Comment:
Given that this icon is a `mailto` link, let's switch to an envelope / email icon.